rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  attr :date, :book, :person
```

Comment:
- [ ] You are expected to have both **_Setters_** and **_getters_** for instance variables (remember about attr_accessor).
This `attr :date, :book, :person` is similar to `attr_reader`, which is only **_getter_**. 

```ruby
attr_accessor :date, :book, :person
```